action.yml

```diff
@@ -41,18 +41,22 @@ inputs:
         Passing multiple package names is allowed.
         Package names can be separated by any whitespace character.
     required: true
-  source-ros-binary-installation:
+  target-ros1-distro:
     description: |
-        Source setup.sh from one or more binary ROS installations (Linux only).
-        For instance, if dashing is passed, /opt/ros/dashing/setup.sh will
-        be sourced before running colcon.
-        Multiple ROS distributions separated by one or more whitespace\
-        characters are allowed. This is typically used to setup a ROS and
-        a ROS 2 distribution to build packages relying on both, such as
-        the ros1_bridge.
-        Examples of valid values include "", "dashing", "melodic dashing".
+        ROS 1 distribution to target for the build.  Will be used both for
+        `rosdep install` and `source setup.bash`. Either or both of
+        `target-ros1-distro` and `target-ros2-distro`, but at least one must
+        be specified. Typically, both are specified when building packages
+        that rely on both ROS versions, such as the ros1_bridge.
+    required: false
```

Comment:
If feels strange / confusing that at least one must be specified, but neither are marked as required. Is there a better to clarify this?

Reply:
There is not advanced argument specification. We must make the check at runtime - it fails fast with an error message if neither is defined.